Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContactActivity.java

```diff
@@ -119,7 +118,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     public void createCode() {
 
-        ImageView qrCodeView = findViewById(R.id.im);
```

Comment:
is this not needed?

Reply:
no, don't believe so. It is declared at the top with BindView, I can do some more testing over the weekend and create a second account to test the QR codes before merging if you would like.

Reply:
just test if this works fine 👍

Reply:
![travel_mate_test](https://user-images.githubusercontent.com/38079783/70381468-b84a2d80-1918-11ea-8ed8-7cf55a28230a.jpg)
I created a second account using an emulator and was able to scan the QR code using my mobile device, everything seems to be working.